Core/DTO/TripDTO/RoutePreviewDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Core.DTO.TripDTO
+{
+    public class RoutePreviewDTO
```

Comment:
For what reason we create another DTO for routes, if we already have RouteDTO with same fields?